subsys/bluetooth/host/Kconfig

```diff
@@ -82,7 +82,7 @@ config BT_RECV_IS_RX_THREAD
 
 config BT_RX_STACK_SIZE
 	int "Size of the receiving thread stack"
-	depends on BT_HCI_HOST || BT_RECV_IS_RX_THREAD
+	depends on BT_RECV_IS_RX_THREAD
 	default 512 if BT_HCI_RAW
 	default 2048 if BT_MESH
 	default 2200 if BT_SETTINGS
```

Comment:
Shouldn't we add/move these defaults for `BT_MESH` and `BT_SETTINGS` to `SYSTEM_WORKQUEUE_STACK_SIZE`? 

Have this change been tested with BT_SETTINGS enabled or with MESH?

Reply:
First of all, this `BT_RX_STACK_SIZE` still needs to be used by the soft based controller, so we can't remove it.

Reply:
Added.